### PR TITLE
Targeting TFM to 7.0

### PR DESCRIPTION
**Pull request by dreddy-work**
contributes to #6323

Upgrading TFM for WinForms main branch to 7.0.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6370)

### CODE REVIEW

**Review by mmitche — APPROVED** [review]

**Comment by dreddy-work** [comment]
Seems intelisense for 7.0 version not published. We may not be able to remove 6.0 feeds until then.

**Comment by dreddy-work** [comment]
waiting on https://github.com/dotnet/core-eng/issues/15222

File: src/System.Windows.Forms/tests/IntegrationTests/System.Windows.Forms.IntegrationTests.Common/TestHelpers.cs [hunk 1]
```diff
@@ -31,7 +31,18 @@ private static string Config
...
         private static string TargetFramework
-            => "net6.0";
+        {
+            get
+            {
+                var frameworkAttribute = (Runtime.Versioning.TargetFrameworkAttribute)Assembly.GetExecutingAssembly()
+                    .GetCustomAttributes(typeof(Runtime.Versioning.TargetFrameworkAttribute), false)
+                    .SingleOrDefault();
+
+                var etractTokens = frameworkAttribute.FrameworkName.Split("=v"); // "NetcoreApp, Version=v7.0"
```
Comment (gpetrou) [inline src/System.Windows.Forms/tests/IntegrationTests/System.Windows.Forms.IntegrationTests.Common/TestHelpers.cs:41]: extractedTokens?

**Pull request merged**